src/components/FeatureViews/Accounts/ProfileHeading.js

```diff
@@ -99,214 +113,343 @@ const GET_PROFILE_HEADING_QUERY = {
-					</>
-				);
-		}
-	};
+  const handleCloseReset = () => {
+    setResetOpen(false);
+  };
+
+  const isAdmin = loggedInAuth.accountType === USER_TYPES.admin;
+  const isUser = loggedInAuth.user.id === userInfo.user.id;
+
+  const renderEditandAwayButton = () => (
+    <Grid container item xs={4}>
+      {(isAdmin || isUser) && (
+        <>
+          <Grid component={Hidden} item mdDown xs={12}>
+            <EditIcon className="editIcon" />
+            <div className="editResetDiv">
+              <ResponsiveButton
+                component={Link}
+                to={`/form/${userInfo.accountType.role}/${userInfo.user.id}`}
+                className="edit"
+              >
+                Edit Profile
+              </ResponsiveButton>
+              {isAdmin &&
+                (userInfo.accountType.role === "student" ? (
+                  <ResponsiveButton
+                    className="resetStudent"
+                    disabled
+                    onClick={handleClickOpen}
+                  >
+                    Reset Password
+                  </ResponsiveButton>
+                ) : (
+                  <ResponsiveButton className="reset" onClick={handleClickOpen}>
+                    Reset Password
+                  </ResponsiveButton>
+                ))}
+            </div>
+            <ResetDialog />
+          </Grid>
+          <Grid component={Hidden} item lgUp xs={12}>
+            <Button
+              component={Link}
+              to={`/form/${userInfo.accountType.role}/${userInfo.user.id}`}
+              variant="outlined"
+            >
+              <EditIcon />
+            </Button>
+          </Grid>
+        </>
+      )}
+    </Grid>
+  );
+
+  const ResetDialog = () => (
+    <>
+      <Dialog
+        open={open}
+        onClose={handleClosePassword}
+        maxWidth="xs"
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogTitle id="alert-dialog-title" className="center dialog-padding">
+          {"Do you want to reset this user's password?"}
+        </DialogTitle>
+        <DialogContent>
+          <DialogContentText id="alert-dialog-description" className="center">
+            When you reset the password, a randomly generated password will
+            become the user's new password. This action cannot be reverted. An
+            automatic email will be sent out notifying the user that their
+            password has been reset. The user can also reset their own password
+            through the portal login page.
+          </DialogContentText>
+        </DialogContent>
+        <DialogActions className="dialog-actions">
+          <Button
+            onClick={handleClosePassword}
+            variant="outlined"
+            className="cancelBtn"
+          >
+            cancel
+          </Button>
+          <Button
+            onClick={handleClosePasswordReset}
+            className="resetBtn"
```

Comment:
I skimmed through the scss so I didn't notice this but, if "resetBtn" class is dictating the background color, you should take that out and use `<Button color="primary"/>` 

Also remember to use `<ResponsiveButton color="primary"/>` instead of Button (both lines 244 and 251)

Reply:
The background color for resetBtn is a light red in the design, not blue. Should I leave it, or change it to blue?

Reply:
Resolved with leaving color as red in teams!